### ISSUE TITLE
Minimum version requirement failed. The minimum CLI version requirement is 3.0.0, your current CLI version is 2.25.2. Please update the Pulumi CLI.

### ISSUE THREAD

**Issue by ffMathy** [pulumi/actions#196]
I'm using V3 of the actions. I don't get why this is failing.

**Comment by stack72** [comment]
Hi @ffMathy 

It looks like you are installing Pulumi CLI v2.25.2 - the v3 of the action is documented in our CHANGELOG of needing at least Pulumi 3.0

I'm not sure if you are using our action to install the CLI or not as part of your actions workflow

Something like:

```
- uses: actions/checkout@v2
- uses: pulumi/action-install-pulumi-cli@v1 <--------- this will install Pulumi CLI latest
- uses: pulumi/actions@v2
  with:
    command: preview
    stack-name: dev
  env:
    PULUMI_ACCESS_TOKEN: ${{ secrets.PULUMI_ACCESS_TOKEN }}
```

**Comment by stack72** [comment]
Closed via #211 

We will handle the usage of the CLI internally and not require to need to think about the CLI
